app.py

```diff
@@ -6,11 +6,14 @@
 # coding=utf-8
 import datetime
 import collections
+from functools import wraps
```

Comment:
I think we're not using this anymore, right?

Reply:
Fixed!